examples/starrocks/deploy_a_ha_starrocks_cluster.yaml

```diff
@@ -3,7 +3,7 @@
 # leader, and the other two are the followers.
 # the number of BEs required depends on the default `replication_num` which is `3` by default.
 # StarRocks cluster.
-# The requests and limits of FE and BE are set based on https://docs.starrocks.io/en-us/latest/deployment/plan_cluster
+# The requests and limits of FE and BE are set based on https://docs.starrocks.io/docs/deployment/plan_cluster/
```

Comment:
Does this mean all links to SR should be updated?

Reply:
any links to `en-us` are from the old doc system. I fixed all of the ones in the examples dir, but I did not check the rest of the dirs. I will do that in a separate PR.